### ISSUE TITLE
Gathering feedback on time period selection tool.

### ISSUE THREAD

**Issue by noahmanger** [18F/FEC#71]
Placeholder issue for the new designs.

**Comment by jenniferthibault** [comment]
We are exploring several options for how to select the amount of time that the data will span on candidate pages. The following designs use a Senate election as an example, but the concepts will also be applied to Presidential and House election candidates.

We welcome public feedback on each option, please feel free to leave your thoughts in the comment fields below.

### Option A: Single drop-down
The user could select two year periods within each election cycle, or the full cycle for each election this candidate  ran in.

![candidate - senate copy](https://cloud.githubusercontent.com/assets/11636908/10526094/bfacaa92-7356-11e5-984a-882a0bb04157.png)

### Option B: Two steps, drop-down and toggle
The first drop-down shows each election the candidate ran in, and the second toggle button breaks that election cycle down to be viewable by two-year periods or the whole cycle.

![candidate - senate copy 2](https://cloud.githubusercontent.com/assets/11636908/10526105/cbe963a4-7356-11e5-85a2-474a9cf22800.png)

### Option C: Two drop-downs
The first drop-down shows each election the candidate ran in, and the second drop-down breaks it down by two-year periods or the whole election cycle.

![candidate - senate copy 3](https://cloud.githubusercontent.com/assets/11636908/10526106/cbee51b6-7356-11e5-879c-51e032d56b84.png)

### Option D: Two steps, drop-down and slider
The first drop-down shows each election the candidate ran in. The slider is updated by the election chosen in the dropdown, and then allows the user to select any amount of time within that election, broken down by quarters.

![candidate - senate copy 4](https://cloud.githubusercontent.com/assets/11636908/10526108/cc01e078-7356-11e5-99f3-445c74b93966.png)

**Comment by evankroske** [comment]
Sliders are hard to use:

http://www.nngroup.com/articles/gui-slider-controls/

**Comment by jenniferthibault** [comment]
Thank you @evankroske — I think we share some of these concerns about sliders. While we figured out what needed to go into this issue, we kind of made a mess of it, so we made a new cleaner one over in # 73 where folks who are newer to GitHub might feel less overwhelmed. Closing this now, but feel free to repost your link over in the new issue (where feedback will be collected from here out) if you'd like!
